commit 42d3987d08e41723a2ca8739550bbf5d58e45ea8
Author: Baptiste Jonglez <git@bitsofnetworks.org>
Date:   Sat Jul 7 12:58:54 2018 +0200

    Disable screen clearing
    
    This tends to fail when building in a clean chroot (e.g. yodl) because of
    missing terminal definition, and is of dubious interest anyway.

diff --color -aur icmake-9.03.01-old/icmake/scripts/ib/main icmake-9.03.01-new/icmake/scripts/ib/main
--- icmake-9.03.01-old/icmake/scripts/ib/main	2020-01-13 11:30:09.000000000 +0200
+++ icmake-9.03.01-new/icmake/scripts/ib/main	2020-08-10 15:10:44.705348577 +0300
@@ -34,7 +34,7 @@
         g_option = _c;
     #endif
 
-    if (g_option == _c)             // clear the screen before compilation
+    if (0)                          // clear the screen before compilation
         system("tput clear");       // starts
 
                                     // compile all sources except MAIN
